packages/cells/style/widget.css

```
@@ -71,11 +71,51 @@
 .jp-CodeCell.jp-mod-outputsScrolled .jp-Cell-outputArea {
   overflow-y: auto;
   max-height: 24em;
-  box-shadow: inset 0 0 6px 2px rgba(0, 0, 0, 0.3);
+  /*box-shadow: inset 0 0 6px 2px rgba(0, 0, 0, 0.3);*/
   margin-left: var(--jp-private-cell-scrolling-output-offset);
   padding-top: 6px;
 }
```

let's try to move the padding to the child

```suggestion
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-OutputArea-child {
  padding-top: 6px;
}
```